akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -151,11 +118,9 @@ object Sink extends SinkApply {
    * element is dropped and the stream continues.
    *
    * @see [[#mapAsyncUnordered]]
-   */
+   */ // FIXME Why is this method even useful?
```

Comment:
I guess the downside is that using this means that it's hard to do an orderly shutdown because the termination of foreachParallel does not imply that the things being `run()` have terminated?